backend/molgenis-emx2-sql-it/src/test/java/org/molgenis/emx2/sql/AToolToCleanDatabase.java

```diff
@@ -0,0 +1,64 @@
+package org.molgenis.emx2.sql;
+
+import static org.jooq.impl.DSL.name;
+
+import java.util.List;
+import org.jooq.DSLContext;
+import org.jooq.ForeignKey;
+import org.jooq.Schema;
+import org.jooq.Table;
+
+public class AToolToCleanDatabase {
```

Comment:
add comment of readme docs on function and usage of tool

Reply:
https://github.com/molgenis/molgenis-emx2/blob/master/DEVELOP.md
see bottom
Maybe hard to find ;-)